paybybank/src/main/java/com/adyen/checkout/paybybank/DefaultPayByBankDelegate.kt

```diff
@@ -201,6 +201,9 @@ internal class DefaultPayByBankDelegate(
 
     override fun isConfirmationRequired(): Boolean = _viewFlow.value is ButtonComponentViewType
 
+    // Pay by bank should not show a button as the entries navigate to the next step.
+    override fun shouldShowSubmitButton(): Boolean = false
```

Comment:
Why can't we also use `isConfirmationRequired() && componentParams.isSubmitButtonVisible` here? The view type can either be null or `PayByBankComponentViewType` which is not a `ButtonComponentViewType`, so the result will be always false